### ISSUE TITLE
Confusion on what to expect out of the box

### ISSUE THREAD

**Issue by mcmurter** [the-couch/slater#56]
I am a little confused on what to expect from slater right out of the box. Perhaps the documentation could make this a little more clear. After a `slater init` I would expect the theme to be in a buildable state but I had to enable SCSS in the config before I was able to build the theme without error. Once built, uploaded and synced the theme seemed to be completely unstyled but by including `main.css` in `theme.liquid` _some_ style was applied (orange background on the body) but it still seems pretty unstyled. I haven't seen any mention of `main.css`, so I am just a little confused. Also without a screenshot to reference I really don't know if I have everything set up properly -- it looks broken but I have seen mention that the starter theme needs some love so that could be the case. I also understand that this is a bit of an opinionated start, but it would be helpful to understand exactly which opinions have been implemented. I am sure many people would love to use slater and build a theme truly from scratch without the opinionated starting point.

I have read [this](https://medium.com/the-couch/getting-started-with-slater-bundling-and-deployment-with-any-existing-shopify-theme-d994a17f590f) blog post, but that is all I can really find as far as any documentation is concerned.

As an aside, kudos and thanks for this project. It is good to see the community building tools as it seems that Shopify can't be bothered to build, maintain and support anything official. I first got really exited about slate at Shopify's Unite 2018 conference and am incredibly disappointed that they have seemingly abandoned the project.

**Comment by iamkevingreen** [comment]
@mcmurter This project is a sort of constant work in progress. Eric and I have built a bunch of Shopify sites, had issues with the tooling over the last few years and as a result this was born. It's been rewritten a number of times. It is by no means some perfect entity 😬 

As a result it's sort of a bit of use at your own risk? At the end of the day the theme will deploy, do code pushes, give you some bundled css/javascript. We try to impose less strict practices vs Slate but at the end of the day it's not for everyone. It's not that beginner friendly and occasionally requires some hacking especially if you are just starting from a recent update. 

All these tools are supposed to make our lives easier or open the doors for better tools. We welcome forks and spinoffs, if you find parts of it useful and others not (some people don't like our choice to go postcss by default for example).

(as for the main.css we've changed the preferred file from main/index a few times apologies there). I'm working on updating this a bit over the next few days and been merging in a few things that have been pending in PRs. 

At the end of the day I can't work on this full time and have to prioritize client work but whenever I start new Shopify sites, I pour a bunch of love into this!
